airflow/utils/orm_event_handlers.py

```diff
@@ -32,6 +32,9 @@ def setup_event_handlers(engine):
     """
     Setups event handlers.
     """
+
+    idle_session_timeout = conf.getint("core", "sql_alchemy_idle_transaction_timeout", fallback=10)
```

Comment:
I didn't add this to airflow.cfg as I was thinking this was a hidden/advanced setting. WDYT?

Reply:
So I will know about it only if I know the code or stumble upon it in some Stackoverflow post 😄 I would be in favor of adding it to airflow.cfg

Reply:
Fair.

Reply:
See my other comment - I believe it (might)  have a profound impact on how MySQL server behaves (especially) because effectively each client will be - in some cases -  continuously closing and opening the connections, where previously they were opened for up to 8 hours by default. So we REALLY need to describe it and I think we have to understand  the impact and describe it also for the MySQL users who would like to enable it. I guess setting it by default (like it is currently) to a low value is rather a bad idea  in those cases so quite a bit more complex logic shoudl be implemented here  - for example it could be set to a low value only when HA scheduler is enabled (when we get to take a look at it).